src/dotnet-install.ps1

```diff
@@ -721,8 +749,7 @@ New-Item -ItemType Directory -Force -Path $InstallRoot | Out-Null
 $installDrive = $((Get-Item $InstallRoot).PSDrive.Name);
 $diskInfo = Get-PSDrive -Name $installDrive
 if ($diskInfo.Free / 1MB -le 100) {
-    Say "There is not enough disk space on drive ${installDrive}:"
-    return
+    throw "There is not enough disk space on drive ${installDrive}:"
```

Comment:
Should we use 
```
Say-Error "There is not enough disk space on drive ${installDrive}:"
throw
```
here?

Reply:
No, Say-Error is for an additional message in case the exception itself might be not clearly stating the problem. We need to go out of the script with an exception.